### DISCUSSION TITLE
ZBM has very small font on 4K screen.

### DISCUSSION THREAD

**Discussion by mitosu-me** [zbm-dev/zfsbootmenu#304] [General]
Is there a way to increase font size with terminus-font?

**Comment by zdykstra** [comment]
If you're using the release binaries, you should be able to pass `rd.vconsole.font=ter-132n` to the ZBM command line and have it set your font size.

**Comment by mitosu-me** [comment]
> If you're using the release binaries, you should be able to pass `rd.vconsole.font=ter-132n` to the ZBM command line and have it set your font size.

How do drop into ZBM command line? Is ctrl+k?

**Reply by zdykstra** [reply]
`CTRL+K` is for changing the command line for the boot environment. Changing the command line for ZBM depends on your stage 1 bootloader. If you're using rEFInd, you can press F2 on the rEFInd loading screen to edit it.

**Reply by mitosu-me** [reply]
Thanks.

**Comment by ahesford** [comment]
If you are booting the ZBM EFI directly instead of using rEFInd, download https://github.com/zbm-dev/zfsbootmenu/blob/master/bin/zbm-efi-kcl and run

    zbm-efi-kcl -e /path/to/your/zfsbootmenu.EFI

and add

    rd.vconsole.font=ter-132n

to the line, save and quit.
